### PR TITLE
Consolidation Of Legal Copy

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

There are some T&Cs tweaks I need to make to the site, and while doing that the branch morphed into a bit of a refactor of the way our T&C helpers and components currently work. I've hopefully made things a bit simpler and improved the semantics a bit.

[**Trello Card**](https://trello.com/c/HGvhs4aM/1111-refactor-tcs-and-footer)

## Changes

- Pulled the various footers into a single file, with flags to enable different content.
- Updated the `TermsPrivacy` component so that it handles the links itself, just pass the country.
- Moved the `legal` helper out of the `internationalisation` folder (because it handles more stuff than just that), and added the copyright notice.
- Replaced footers throughout the codebase with the new one.

## Screenshots

**Copyright Only:**

just-copyright

**Copyright And Link:**

copyright-and-link

**Copyright And Disclaimer:**

copyright-and-disclaimer

**All The Things:**

all-the-things

### CODE REVIEW

File: assets/pages/bundles-landing/bundlesLanding.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const content = (
       <Bundles />
       <WhySupport />
       <WaysOfSupport />
-      <LinksFooter />
+      <Footer privacyPolicy />
```
Comment (svillafe) [inline assets/pages/bundles-landing/bundlesLanding.jsx]: Where is `privacyPolicy` defined? I can't see it in the imports :-/

Comment (Ap0c) [reply assets/pages/bundles-landing/bundlesLanding.jsx]: It's not, that's a boolean.

Comment (svillafe) [reply assets/pages/bundles-landing/bundlesLanding.jsx]: Is it possible to do something more readable like `privacyPolicy = true`?

Comment (Ap0c) [reply assets/pages/bundles-landing/bundlesLanding.jsx]: That causes a lint error, it tells you to drop the `true`.

**Pull request merged**